contribs/gnogenesis/internal/txs/txs_add_packages.go

```diff
@@ -117,3 +131,55 @@ func execTxsAddPackages(
 
 	return nil
 }
+
+func signTxs(txs []gnoland.TxWithMetadata, signer *gnoclient.SignerFromKeybase) error {
+	for index, tx := range txs {
+		signBytes, err := tx.Tx.GetSignBytes(signer.ChainID, 0, 0)
+		if err != nil {
+			return fmt.Errorf("unable to load txs from directory, %w", err)
+		}
+		signature, publicKey, err := signer.Keybase.Sign(signer.Account, signer.Password, signBytes)
+		txs[index].Tx.Signatures = []std.Signature{
+			{
+				PubKey:    publicKey,
+				Signature: signature,
+			},
+		}
+		if err != nil {
+			return fmt.Errorf("unable sign tx %w", err)
+		}
+	}
+
+	return nil
+}
+
+func signerWithConfig(cfg *addPkgCfg, io commands.IO, chainID string) (*gnoclient.SignerFromKeybase, error) {
+	var (
+		keyname = integration.DefaultAccount_Name
```

Comment:
We shouldn't use the integration account name and seed -- you can freely redefine it in this package

Reply:
Done on 09dce4322f68e8f7aa366c3c33da287d5761d8ae

Reply:
They don't need to be exported 🙏